package/MDAnalysis/analysis/base.py

```diff
@@ -169,12 +171,15 @@ def run(self, start=None, stop=None, step=None, verbose=None):
             number of frames to skip between each analysed frame
         verbose : bool, optional
             Turn on verbosity
+            
+        .. versionchanged:: 1.0.0
+        verbose explicitly passed to _setup_frames as argument for temporary 
```

Comment:
Same as above, please do use :meth: and :class: docstring decorators too if you can, that way users can navigate to the right place when looking at the sphinx documentation.